### ISSUE TITLE
lucky db.schema.dump does not show help message when argument is omitted

### ISSUE THREAD

**Issue by grepsedawk** [luckyframework/avram#773]
![image](https://user-images.githubusercontent.com/6721355/147400260-e7ebd33b-0f2f-4742-9e91-d78679194bcf.png)

When no arg is passed to db.schema.dump, it results in a fatal error.
It seems like there is intended to be an "uhoh" message that never shows up, here https://github.com/luckyframework/avram/blob/master/src/avram/tasks/db/schema_dump.cr#L11-L19

**Comment by grepsedawk** [comment]
Related to lucky#1463

**Comment by jwoertink** [comment]
Good catch. That code you linked to is actually the help message when you run `lucky db.schema.dump -h`.

This error message comes from
https://github.com/luckyframework/lucky_task/blob/da2bfd5d69c6aab7e5c4a1e998469645d4b31a63/src/lucky_task/task.cr#L106

We definitely need to clean that up.
